x-pack/plugin/security/src/main/java/org/elasticsearch/xpack/security/authc/saml/SamlRequestHandler.java

```diff
@@ -141,14 +143,20 @@ void validateSignature(Signature signature) {
         }
 
         checkIdpSignature(credential -> {
-            try (RestorableContextClassLoader ignore = new RestorableContextClassLoader(SignatureValidator.class)) {
-                SignatureValidator.validate(signature, credential);
-                logger.debug(() -> new ParameterizedMessage("SAML Signature [{}] matches credentials [{}] [{}]",
-                        signatureText, credential.getEntityId(), credential.getPublicKey()));
-                return true;
+            try {
```

Comment:
This is rather peculiar, but I can't figure out why the doPrivileged block is necessary _now_ when the cluster is run in our QA project

Reply:
We discussed on Zoom. The RestorableContextClassLoader was moved from security to core in the IDP branch, but it looks like all the tests that cover this are disabled.